dev-qt/qtwebengine/qtwebengine-5.15.2_p20211210.ebuild

```diff
@@ -110,29 +110,47 @@ PATCHES=(
 	"${WORKDIR}/${PN}-5.15.2_p20211019-jumbo-build.patch" # bug 813957
 )
 
-pkg_preinst() {
-	elog "This version of Qt WebEngine is based on Chromium version 87.0.4280, with"
-	elog "additional security fixes from newer versions. Extensive as it is, the"
-	elog "list of backports is impossible to evaluate, but always bound to be behind"
-	elog "Chromium's release schedule."
-	elog "In addition, various online services may deny service based on an outdated"
-	elog "user agent version (and/or other checks). Google is already known to do so."
-	elog
-	elog "tldr: Your web browsing experience will be compromised."
-}
-
-src_unpack() {
-	# bug 307861
+qtwebengine_check-reqs() {
+	# bug #307861
 	eshopts_push -s extglob
 	if is-flagq '-g?(gdb)?([1-9])'; then
-		ewarn
 		ewarn "You have enabled debug info (probably have -g or -ggdb in your CFLAGS/CXXFLAGS)."
 		ewarn "You may experience really long compilation times and/or increased memory usage."
 		ewarn "If compilation fails, please try removing -g/-ggdb before reporting a bug."
-		ewarn
 	fi
 	eshopts_pop
 
+	[[ ${MERGE_TYPE} != binary ]] || return
+
+	# (check-reqs added for bug #570534)
+	#
+	# Estimate the amount of RAM required
+	# Multiplier is *10 because Bash doesn't do floating point maths.
+	# Let's crudely assume ~2GB per compiler job for GCC.
+	local multiplier=20
+
+	# And call it ~1.5GB for Clang.
+	if tc-is-clang ; then
+		multiplier=15
+	fi
+
+	local CHECKREQS_DISK_BUILD="7G"
+	local CHECKREQS_DISK_USR="7G"
```

Comment:
uhm really 7 GB of installed size? I don't currently have qtwebengine installed so cannot quickly check... can anyone confirm?

Reply:
```
# equery s qtwebengine
 * dev-qt/qtwebengine-5.15.2_p20211216
         Total files : 263
         Total size  : 149.60 MiB
```

Reply:
yeah, seems obviously wrong, will fix